oaknational/Oak-Web-Application
# [PUPIL-771] Add pupil pages to Percy and Pa11y

## Description

Added the following urls to Pa11y and Percy

- /pupils/years
- /pupils/years/year-10/subjects
- /pupils/programmes/combined-science-secondary-year-10/options
- /pupils/programmes/maths-primary-year-5/units
- /pupils/programmes/maths-primary-year-5/units/multiplication-by-partitioning-leading-to-short-multiplication-3-by-1-digit/lessons
- /pupils/programmes/geography-primary-year-5/units/north-and-south-america-how-diverse-are-their-places-and-landscapes/lessons/physical-geography-of-north-and-south-america/overview
- /pupils/programmes/computing-primary-year-5-l/units/vector-drawing-ea06/lessons/create-a-vector-drawing-cgvpac/intro

Additionally I added a one line fix for a Pa11y issue ticketed [here](https://www.notion.so/oaknationalacademy/a11y-Remove-redundant-list-item-role-from-lesson-list-component-lesson-listing-page-d33ee2b089804c5381e29d4763b27483) where there were nested `list-item` aria roles.

## How to test

1. Go to https://deploy-preview-2508--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-primary-year-5/units/multiplication-by-partitioning-leading-to-short-multiplication-3-by-1-digit/lessons
2. Pa11y issue should be resolved. 

## Checklist
